packages/builtin/string.pony

```diff
@@ -611,7 +611,7 @@ actor Main
     _ptr._copy_to(str._ptr._unsafe(), len)
     str._size = len
     str._set(len, 0)
-    str
+    consume str
```

Comment:
The last line of a function should be consumed automatically when needed (at the language level), so I'm not sure why this change was done.

Reply:
Ah, I recalled something like that. What was happening before was that because of the unsound subtyping checks, it  didn't matter whether the last line was consumed or not. I don't know that any mechanism was in place to auto-consume

Reply:
I think a lot of user code is relying on the auto-consume like a "feature", so I wouldn't want to roll out this change until we have a way to get that behavior back.

Reply:
Agreed